tests/topotests/ospf_metric_propagation/test_ospf_metric_propagation.py

```diff
@@ -220,10 +220,10 @@ def test_link_1_2_down():
         pytest.skip("skipped because of router(s) failure")
 
     tgen.net["r2"].cmd("ip link set dev r2-eth1 down")
+    tgen.net["r1"].cmd("ip link set dev r1-eth0 down")
```

Comment:
I'm confused why we needed a sleep and no longer do. I still think it is better to check/poll for *some* condition after you interface down to make sure what we think is happening is in fact what *is* happening.

Reply:
As I mentioned in other context before, we shouldn't need a sleep really here but I wasn't getting the behavior I expected due to a bug (feature)? anyhow, I need a trigger for a state change. I initially had a sleep to git rid of a neighbor and add it again to force a recalculation of metrics, but I discovered bringing the interface down/up have the same effect. I'm still working on fixing the underlying  issue of why that is needed, but that is not related to this PR. For now I want to fix this test to avoid having it a friction point in our CI.